cosmwasm/contracts/pyth/src/governance.rs

```diff
@@ -86,13 +97,50 @@ impl GovernanceInstruction {
         let target_chain_id: u16 = bytes.read_u16::<BigEndian>()?;
 
         let action: Result<GovernanceAction, String> = match action_type {
+            0 => {
+                let mut address: [u8; 20] = [0; 20];
+                bytes.read_exact(&mut address)?;
+                Ok(GovernanceAction::UpgradeContract { address })
```

Comment:
I think this is actually a bad idea because it makes it harder to extend the message format later. If we add new fields to the messages, we don't want to force ourselves to update all of the code that reads it simultaneously.